tests/MongoDB.Bson.Tests/Serialization/Serializers/BsonPrimitiveSerializerTests.cs

```diff
@@ -41,6 +40,16 @@ public class TestClass
             public bool S;
         }
 
+        public class TestClassWithPrivate
+        {
+            [BsonRepresentation(BsonType.String)]
+#pragma warning disable CS0649
```

Comment:
Actually constructor is no longer required (I used it in the previous version of the tests). But without it, we need to suspend the warning.

Reply:
OK... I agree that the constructor is not required for the test (at least the way the test is written), but the class sure looks weird without a constructor. A real class would never look like this (there is no way to provide a value for _b).

The fact that you had to suppress a warning is a sign that the class is not "normal".

My vote: leave the constructor as it was and don't have to suppress a weird warning.